tracer/src/Datadog.Trace/IAST/Telemetry/IastInstrumentationMetricsHelper.cs

```diff
@@ -0,0 +1,75 @@
+// <copyright file="IastInstrumentationMetricsHelper.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using Datadog.Trace.ClrProfiler;
+using Datadog.Trace.Telemetry;
+using static Datadog.Trace.Telemetry.Metrics.MetricTags;
+
+namespace Datadog.Trace.Iast.Telemetry;
+
+internal static class IastInstrumentationMetricsHelper
+{
+    private static int _instrumentedPropagations = 0;
+    private static int _sinksCount = Enum.GetValues(typeof(IastInstrumentedSinks)).Length;
+    private static int _sourceTypesCount = Enum.GetValues(typeof(IastInstrumentedSources)).Length;
+    private static int[] _instrumentedSources = new int[_sourceTypesCount];
```

Comment:
Even though we are not yet recording the non callsite instrumentations, the variables to store them are defined and will be filled in the next PR.